[2025-07-24T13:42:04.714+0000] {scheduler_job_runner.py:781} ERROR - Executor reports task instance <TaskInstance: tune_stream_dag.Load_songplays_fact_table scheduled__2023-01-01T20:00:00+00:00 [queued]> finished (failed) although the task says it's queued. (Info: None) Was the task killed externally?
